src/activate.ts

```diff
@@ -27,9 +28,23 @@ async function Activate(): Promise<void> {
                 const servicesConfig = core.getInput('services-config', { required: true });
                 await licenseClient.ActivateLicenseWithConfig(servicesConfig);
             } else {
-                const username = core.getInput('username', { required: true }).trim();
-                const password = core.getInput('password', { required: true }).trim();
-                const serial = core.getInput('serial', { required: license.toLowerCase().startsWith('pro') });
+                const pro = license.toLowerCase().startsWith('pro');
```

Comment:
I don't understand the purpose of splitting this out if we don't use it until several lines later after the if check.